### DISCUSSION TITLE
The layout model is from PP-Structurev2-layout?

### DISCUSSION THREAD

**Discussion by LiZhenzhuBlog** [RapidAI/RapidOrientation#20] [Q&A]
The layout model is from PP-Structurev2-layout?  or the PP-Structurev1  ?

**Comment by SWHL** [comment]
From this [link](https://github.com/PaddlePaddle/PaddleOCR/blob/133d67f27dc8a241d6b2e30a9f047a0fb75bebbe/ppstructure/layout/README_ch.md), I am not clear what is PP-StructureV2 or V1.
